cad/gtkwave/Portfile

```diff
@@ -40,9 +40,6 @@ configure.args-append \
                 --with-tcl=${prefix}/lib \
                 --with-tk=${prefix}/lib \
                 --disable-mime-update \
-                --disable-silent-rules \
```

Comment:
this should remain disabled - or is it not a configure argument anymore?

Reply:
I added back the disable-silent-rules.